### DISCUSSION TITLE
Docker image for arm64

### DISCUSSION THREAD

**Discussion by DwarakJ** [Budibase/budibase#4292] [Help] [answered]
I tried to host budibase in Mac M1, budi hosting failing because it couldn't find the matching manifest for linux/arm64/v8. Can we build a image for arm64.

**Comment by shogunpurple** [comment, marked as answer]
Hi @DwarakJ - you should be able to run budibase on an M1 mac - I use one myself.

This is mentioned in the docs here: https://docs.budibase.com/self-hosting/self-host/budibase-cli-setup#running-budibase

You need:

```
 couchdb-service:
    platform: linux/amd64
    ...
  
```
